data_generation/download_solar_omni.py

```diff
@@ -0,0 +1,259 @@
+import numpy as np
+import pandas as pd
+import datetime
+from geospacepy import omnireader
+from sunpy.net import hek, Fido, attrs as a
+from sunpy.timeseries import TimeSeries
+import matplotlib.pyplot as plt 
+import csv
+
+def download_omni_text(input_datetime, file_path=os.getwcd()+'/omni_data/'):
```

Comment:
Should that be `os.getcwd()`? 

Also, I've found using the `Path` module from `pathlib` useful:
```

import os
from pathlib import Path

root_folder = Path(os.getcwd()).parent.parent 
data_root = os.path.join(root_folder, 'data')

```
etc.